doc/API.rst.txt

```diff
@@ -150,6 +150,31 @@ change is made.
   - `scipy.stats.mstats`
   - `scipy.stats.qmc`
 
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   cluster
+   constants
+   fft
+   fftpack
+   integrate
+   interpolate
+   io
+   linalg
+   misc
+   ndimage
+   odr
+   optimize
+   signal
+   sparse
+   sparse.linalg
+   sparse.csgraph
+   spatial
+   special
+   stats
+   stats.mstats
```

Comment:
It's weird that you have `stats.mstats` here but not `stats.qmc` or `stats._result_classes`, maybe that's the problem?

Reply:
(based on `doc/source/stats.rst` in `master` it's what I would have expected)

Reply:
mmm I might have wrongly copied over things. Thanks I will check.

Reply:
Ah ok this part I actually copied from the original `index.rst`. This was required to have these files included in a toc tree.